aya/src/programs/utils.rs

```diff
@@ -51,3 +60,41 @@ pub(crate) fn find_tracefs_path() -> Result<&'static Path, ProgramError> {
         .as_deref()
         .ok_or_else(|| io::Error::new(io::ErrorKind::Other, "tracefs not found").into())
 }
+
+/// Get the program's load time.
+///
+/// The load time is specified by the kernel as nanoseconds since system boot,
+/// this function converts that u64 value into a human readable string.
+pub(crate) fn print_load_time(ns_since_boot: u64) -> String {
```

Comment:
Changing from the stdlibs Duration/SystemTime should happen outside of Aya. This function should only return a stdlib time type.

Reply:
Done! Returned a SystemTime here.